runtimes/data-binding.mdx

```diff
@@ -773,11 +774,9 @@ Property descriptors can be inspected on a view model to discover at runtime whi
     </Tab>
 </Tabs>
 
-### Mutable Properties
+References to these properties can be retrieved by name or path. Mutable properties have getters, setters, and observe operations for their values, while other properties (except nested view models) only have observe operations. Getting or observing the value will retrieve the latest value set on that properties binding, as of the last state machine or artboard advance. Setting the value will update the value and all of its bound elements.
 
-View model instances have mutable properties. References to these properties can be retrieved by name. They have get, set, and observe operations. Getting or observing the value will retrieve the latest value set on that properties binding, as of the last state machine or artboard advance. Setting the value will update the value and all of its bound elements.
-
-<Note>Trigger properties do not have a get operation - only set and observe.</Note>
+<Note>Trigger properties do not have a get or set operation - only trigger and observe.</Note>
```

Comment:
Maybe we don't even need this note. The API for data binding is expanding and there are more conditionals now, and each property can have a unique api